local/php/envs.go

```diff
@@ -37,11 +37,30 @@ func (p *Server) generateEnv(req *http.Request) map[string]string {
 	}
 
 	pathInfo := req.URL.Path
+	filenameMatch := false
+
 	if pos := strings.Index(strings.ToLower(pathInfo), ".php"); pos != -1 {
 		file := pathInfo[:pos+4]
 		if _, err := os.Stat(filepath.Join(p.documentRoot, file)); err == nil {
 			scriptName = file
 			pathInfo = pathInfo[pos+4:]
+			filenameMatch = true
+		}
+
+	}
+
+	if pathInfo != string(os.PathSeparator) && filenameMatch == false {
+		paths := strings.Split(strings.TrimRight(pathInfo[1:], string(os.PathSeparator)), string(os.PathSeparator))
```

Comment:
we are not dealing with filepaths but with URLs here, so we need to split by `/` instead of `os.PathSeparator`

Reply:
`Trim` instead of `TrimRight` is also probably necessary

Reply:
Actually it's weird, but changing to trim might actually cause some damage by stripping more slashes from the front which can be problematic. 

Basically, it should work like that:
```
/subdirectory/ -> subdirectory/index.php
//subdirectory/ -> index.php
```

I guess the more common behaviour of apache or nginx would resolve it to root too as // wouldn't be found as a folder and it's probably better that user would figure out their URLs are wrong locally than when deployed. 

However, should definitely add a test case for that.

Reply:
Goddamit, sorry. I am thinking out loud apparently. Seems that default behaviour when working with paths like `//subidrectory//index.php/something` is actually a bit inconsistent currently, so I am gonna keep this same behaviour where basically // is ignored ( said that, passed as query string )